.gitignore

```diff
@@ -6,7 +6,6 @@ yarn-debug.log*
 yarn-error.log*
 lerna-debug.log*
 xata
-schema.json
```

Comment:
Question. Did you remove this for a reason?

Reply:
Yes, because we need [this file](https://github.com/xataio/client-ts/pull/34/files#diff-ddbac6e328728ee2a7a8b36e7f2e73be149d74294c8925fdada37d5d24fd35b2).

Since we ignore the `xata` folder in the line directly above this one, we still meet the need of ignoring generated `schema.json`s while keeping intentional ones for tests and co.